.github/workflows/openmis-module-test.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       - name: install linux packages
         run: |
           git clone --depth 1 --branch develop https://github.com/openimis/database_ms_sqlserver.git  ./sql
-          cd sql/ && bash concatenate_files.sh && cd ..
+          cd sql/ && bash ./scriot/concatenate_files.sh && cd ..
```

Comment:
I think it's a typo. Shouldn't it be script?

Reply:
@delcroip Could you confirm?

Reply:
indeed I will fix it